src/share/classes/com/sun/javatest/regtest/tool/OptionDecoder.java

```diff
@@ -185,12 +186,16 @@ private void decodeArg(String arg, Iterator<String> iter) throws BadArgs {
         if (debugOptions)
             System.err.println("OptionDecoder.decodeArg: " + name + " " + value);
 
-        o.process(arg, value);
+        try {
+            o.process(arg, value);
+        } catch (InvalidPathException e) {
+            throw new BadArgs(i18n, "opt.bad.path.for.option", arg, e.getInput(), e.getMessage());
+        }
     }
 
-    public void addFile(File file) throws BadArgs {
-        fileOption.process(null, file.getPath());
-    }
+//    public void addFile(File file) throws BadArgs {
+//        fileOption.process(null, file.getPath());
+//    }
```

Comment:
Retained intentionally or to-be-converted?

Reply:
... probably, to be removed

Reply:
removed